packages/fluentui/react-northstar/src/components/Form/FormInput.tsx

```diff
@@ -0,0 +1,40 @@
+import { compose, ComponentWithAs, ShorthandConfig } from '@fluentui/react-bindings';
+import { commonPropTypes } from '../../utils';
+import * as customPropTypes from '@fluentui/react-proptypes';
+import Input, { InputProps } from '../Input/Input';
+import FormFieldCustom, { FormFieldCustomProps, FormFieldCustomStylesProps } from './FormFieldCustom';
+
+interface FormInputOwnProps extends Omit<InputProps, 'styles' | 'accessibility'> {}
+
+export interface FormInputProps extends FormFieldCustomProps, FormInputOwnProps {}
+export type FormInputStylesProps = never;
+
+export const FormInputClassName = 'ui-form-input';
+
+const FormInput = compose<
+  'div',
+  FormFieldCustomProps,
+  FormInputStylesProps,
+  FormFieldCustomProps,
+  FormFieldCustomStylesProps
+>(FormFieldCustom, {
+  className: FormInputClassName,
+  displayName: 'FormInput',
+  overrideStyles: false,
```

Comment:
```suggestion
  overrideStyles: true,
```

Reply:
Doing that we miss the styles from `errorMessage`

Reply:
Done